avm/src/daemon/ctrl.rs

```diff
@@ -124,6 +140,66 @@ async fn control_port(req: Request<Body>) -> Result<Response<Body>, Infallible>
   }
 }
 
+async fn handle_start(req: Request<Body>) -> Result<Response<Body>, Infallible> {
+  // Receive POST and save daemon properties
+  match get_daemon_props(req).await {
+    Ok(_) => Ok(Response::builder().status(200).body("ok".into()).unwrap()),
+    Err(err) => {
+      error!(ErrorType::DaemonStartFailed, "{:?}", err).await;
```

Comment:
... we can do something really clever on the `macro_rules! error` so that we can just say `error!(DaemonStartFailed, ...)` ...

see <https://github.com/clap-rs/clap/blob/master/src/macros.rs#L321-L326>:
```rust
macro_rules! error {
  (...) => {
    let err = anycloud::error::ErrorType::$err;
    ...
  }
};
```

Reply:
Like this, will try it out

Reply:
Applied!